#4810-metro-ontime.md-FIXED

joelfuelling opened the pull request:
Fixes #4810

### What changes did you make?
Altered metro-ontime.md tools from string display to array display per gitHub Instructions

### Why did you make the changes (we will use this info to test)?
To complete my 1st 'good issue' from the H4LA Project Board and to complete the task.

Moving files to another directory. No visual changes to the website.

joelfuelling committed:
Need to make branch for this

jnomad21 commented:
Availability: 7/27 Afternoon
Expected Completion: EOD 7/27

jnomad21 reviewed (changes requested):
Hi @joelfuelling overall the changes look good.  You are using the right branch and have the correct linked issue.  The edits look accurate to the request.

The one thing I noticed when looking at the changes is that it appears that you unintentionally have a code formatting tool that is trying to change the formatting of the quotation marks from ' to ''.  Please change those back to keep consistent formatting.  

Can any other PR reviewers comment on how to work around the code formatters so both Joel and I know?

joelfuelling commented:
Hi @***@***.*** ***@***.***>

Great catch! I'll look into my configuration file and see what I can do on
my end to avoid that.

Great catch!
On Thu, Jul 27, 2023, 1:57 PM Jason McGrath ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @joelfuelling <https://github.com/joelfuelling> overall the changes
> look good. You are using the right branch and have the correct linked
> issue. The edits look accurate to the request.
>
> The one thing I noticed when looking at the changes is that it appears
> that you unintentionally have a code formatting tool that is trying to
> change the formatting of the quotation marks from ' to ''. Please change
> those back to keep consistent formatting.
>
> Can any other PR reviewers comment on how to work around the code
> formatters so both Joel and I know?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hackforla/website/pull/5103#pullrequestreview-1550553816>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6LUKSPEI6YXLXNSEDCBRSTXSK22VANCNFSM6AAAAAA22NMBMY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

joelfuelling commented:
@roslynwythe Hi Roslyn

I fixed this issue but my auto-formatting changed some formatting I didn't catch. I deleted my branch after sending it in, whoops! Anyway, I need to re push with the corrected changes. How would I got about doing that since I deleted my branch? 

Thanks!

adrianang commented:
> @roslynwythe Hi Roslyn
> 
> I fixed this issue but my auto-formatting changed some formatting I didn't catch. I deleted my branch after sending it in, whoops! Anyway, I need to re push with the corrected changes. How would I got about doing that since I deleted my branch?
> 
> Thanks!

Hi @joelfuelling! If you deleted your branch from your local machine, try running the git command in [this comment](https://github.com/hackforla/website/pull/5103#issuecomment-1654098144) from the github-actions bot to get back to the commit point that you had made this PR for.

In addition to the change that @jnomad21 had requested, I also noticed that you added a folder `_ANotes` and a file `Roles.md`, which seems like a note you had written for yourself for personal reference. Please delete this from your fork of the Website, as your fork of the website should mimic the production branch except for the feature branch for the issue you're working on (which you make a PR for to merge into production). Let me know or comment on the issue again if you run into any issues!

joelfuelling commented:
Hi Adrian,

Understood! Thanks for the direction, I just pushed the new branch with the
corrected changes.

Hi @***@***.*** ***@***.***>,

This new request should follow the H4LA requirements, but please let me
know if you see otherwise.

Thank you,

On Sat, Jul 29, 2023 at 6:30 AM Adrian Ang ***@***.***> wrote:

> @roslynwythe <https://github.com/roslynwythe> Hi Roslyn
>
> I fixed this issue but my auto-formatting changed some formatting I didn't
> catch. I deleted my branch after sending it in, whoops! Anyway, I need to
> re push with the corrected changes. How would I got about doing that since
> I deleted my branch?
>
> Thanks!
>
> Hi @joelfuelling <https://github.com/joelfuelling>! If you deleted your
> branch from your local machine, try running the git command in this
> comment
> <https://github.com/hackforla/website/pull/5103#issuecomment-1654098144>
> from the github-actions bot to get back to the commit point that you had
> made this PR for.
>
> In addition to the change that @jnomad21 <https://github.com/jnomad21>
> had requested, I also noticed that you added a folder _ANotes and a file
> Roles.md, which seems like a note you had written for yourself for
> personal reference. Please delete this from your fork of the Website, as
> your fork of the website should mimic the production branch except for the
> feature branch for the issue you're working on (which you make a PR for to
> merge into production). Let me know or comment on the issue again if you
> run into any issues!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hackforla/website/pull/5103#issuecomment-1656709785>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6LUKSPXD2WEHC7HAXLHUNDXSTX5NANCNFSM6AAAAAA22NMBMY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
- Joel Fuelling

LRenDO commented:
Availability: 7/31
ETA: 7/31

LRenDO reviewed (commented):
Hi @joelfuelling! Thanks for taking on this issue and working on the requested changes! Since you had to recreate your branch, it looks like now the most current pull request for this issue is #5118. Is that correct? If so, please let @jnomad21 know and add them as reviewer on that pull request, so the changes can be reviewed there.  Thanks!

t-will-gillis commented:
This PR was replaced by #5118
